Update change-log.md

MartaD opened the pull request:
Product update on 08/03/2024

mathieu-benoit commented on docs/change-log.md:
```diff
@@ -2,6 +2,30 @@
+    - `humctl resources` graph to inspect your resource graph.
+  -  `0.14.2` is now installing shell completions on osx by default when using brew for an improved UX.
+
+- **[Humanitec Terraform Provider](https://registry.terraform.io/providers/humanitec/humanitec/latest/docs):** A new version `v1.0.1` has been released, introducting example of a resource definition with secret_refs and private git.
```
```suggestion
- **[Humanitec Terraform Provider](https://registry.terraform.io/providers/humanitec/humanitec/latest/docs):** A new version `v1.0.1` has been released, introducing example of a resource definition with secret_refs and private git.
```

mathieu-benoit commented on docs/change-log.md:
```diff
@@ -2,6 +2,30 @@
+    - `humctl score init` simplifies your next score configuration with a score starter file.
+    - `humctl score validate` to validate your score file locally.
+    - `humctl score available-resource-types` to list available resource types available to be used.
+    - `humctl resources` graph to inspect your resource graph.
```
```suggestion
    - `humctl resources graph` to inspect your resource graph.
```

sujaya-sys commented on docs/change-log.md:
```diff
@@ -2,6 +2,30 @@
 
+## Product Update March 8th, 2024
+
+- **New:** Introduced Deployment Pipelines which are multi-step workflows used to automate common processes and interactions around Applications and Environments. You can find more information in updated [developer documentation](https://developer.humanitec.com/integration-and-extensions/humanitec-pipelines/overview/) and the [release announcement](https://humanitec.com/blog/feature-announcement- ...
```
```suggestion
- **New:** Introduced Deployment Pipelines which allow to configure multi-step workflows and automate common processes and interactions around Applications and Environments. You can find more information in the updated [developer documentation](https://developer.humanitec.com/integration-and-extensions/humanitec-pipelines/overview/) and the [release announcement](https://humanitec.com/blog/feature-announcement-humanitec-deployment-pipelines) on our blog.
```

sujaya-sys commented on docs/change-log.md:
```diff
@@ -2,6 +2,30 @@
+  - `0.14.0` Introducing additional commands:
+    - `humctl score init` simplifies your next score configuration with a score starter file.
+    - `humctl score validate` to validate your score file locally.
+    - `humctl score available-resource-types` to list available resource types available to be used.
```
```suggestion
    - `humctl score available-resource-types` to list available resource types to be used.
```

sujaya-sys commented:
@delca85 @docktorrr would you be able to provide a brief summary of which changes/fixes we're releasing with the

* latest agent update (1.1.7)
* latest operator update (Chart version 0.1.6 and app version 0.10.2)

as it'll be great to include this in the public change log - thank you!!

delca85 commented:
> @delca85 @docktorrr would you be able to provide a brief summary of which changes/fixes we're releasing with the
> 
> * latest agent update (1.1.7)
> * latest operator update (Chart version 0.1.6 and app version 0.10.2)
> 
> as it'll be great to include this in the change log - thank you!! (Feel free to double check on the other changes listed here as well of course)

On the [agent 1.1.7](https://github.com/humanitec/agent-client/releases/tag/1.1.7), we added a check on agent startup which validates the ids in the [ORGS](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/installation/#configuration) list as valid Humanitec ids.

sujaya-sys commented on docs/change-log.md:
```diff
@@ -2,6 +2,30 @@
+
+#### Versioned component changes:
+
+- **[Humanitec Agent](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/installation/):** A new version `1.1.7` has been released.
```
```suggestion
- **[Humanitec Agent](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/installation/):** A new version `1.1.7` has been made available. This release includes a check on agent startup to validate the ids in the [ORGS list](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/installation/#configuration) as valid Humanitec ids.
```

delca85 replied:
This bit LGTM!

docktorrr commented on docs/change-log.md:
```diff
@@ -2,6 +2,30 @@
+- **[Humanitec Agent](https://developer.humanitec.com/integration-and-extensions/humanitec-agent/installation/):** A new version `1.1.7` has been released.
+
+- **[Humanitec Operator](https://developer.humanitec.com/integration-and-extensions/humanitec-operator/installation):** A new version has been published. Chart version `0.1.6` and app version `0.10.2` including cookies secrets storage optimization (including GCP cleanup).
+
```
@sujaya-sys I think this is what the update is about, looks good.

sujaya-sys replied:
cool, thanks for checking! @MartaD FYI, looks like this is good to go :-)

sujaya-sys reviewed (approved):
LGTM

MartaD commented:
> Wondering if we should also add:
> 
> * Even if it's about the docs, I think it would be great to highlight the new Examples Hub library in the docs? WDYT?
> * What about the new private TF runner feature?
> * We also fixed the fact that `base-env` was not listed in the Shared resources list if taking too much time to provision, should we indicate this too?

Thank you Mathieu, Susa and I added them, please review!

mathieu-benoit reviewed (approved):
LGTM
